language/move-prover/bytecode/src/spec_instrumentation.rs

```diff
@@ -978,11 +981,19 @@ impl<'a> Instrumenter<'a> {
 
         // Emit specification variable updates. They are generated for both verified and inlined
         // function variants, as the evolution of state updates is always the same.
-        self.emit_updates(spec, &[]);
+        let lets_emitted = if !spec.updates.is_empty() {
+            self.emit_lets(spec, &[], true);
```

Comment:
I guess there will be conflicts on this once #9183 lands (the `&[]` can be removed, so is the `targs` above). I just realize that I have a habit on stepping others' toes although this is not my intention...

Reply:
Yes, I knew there would be conflicts when seeing the other PR, but no problem, easy to resolve.